yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -151,16 +147,20 @@ def process_video_files(v):
         duration = (int_or_none(try_get(metadata, lambda x: x['content']['duration']))
                     or parse_duration(self._html_search_meta('duration', webpage)))
 
-        description = self._html_search_regex(
-            r'(?s)<div[^>]+class=["\']episode-texte[^>]+>(.+?)</div>', webpage,
-            'description', fatal=False)
+        title = episode = self._html_search_regex(r'<h1 class="main-title">([^<]+)', webpage, 'title')
+        series = self._html_search_regex(r'<p class="video-title">([^<]+)', webpage, 'title', default=None)
+        subtitle = self._html_search_regex(r'<p class="video-subtitle">([^<]+)', webpage, 'subtitle', default=None)
+        if series and subtitle:
+            title = '%s - %s' % (series, subtitle)
+            episode = subtitle
```

Comment:
I know the extractor previously did something similar, but I am not sure if we should touch the title. Maybe just return subtitle always as the episode title, and then the user can use output templates to set the title they want? e.g. `-o "%(series)s - %(episode)s.%(ext)s"`

At the least, change this to use f-strings.